.gitignore

```diff
@@ -8,6 +8,7 @@
 /resources/host/
 /resources/rancher-dashboard/*
 /resources/preload.js*
+/resources/rdx-proxy.tar
 /resources/rdx-proxy.tgz
```

Comment:
The `.tgz` version should no longer exist, then, right?

Reply:
Correct. I kept it because when you switch between branches you get some noise from `git status` until all the branches are the same. I can remove it and ignore the noise; maybe that's cleaner.